src/assets/js/search.js

```diff
@@ -34,7 +34,7 @@ const searchClearBtn = document.querySelector('#search__clear-btn');
  */
 function fetchSearchResults(query) {
     return index.search(query, {
-        // facetFilters: ["tags:docs"]
+        filters: "blog"
```

Comment:
We definitely have content marked with `tag:blog` and `tag:docs`. We are using this in the current site:

https://github.com/eslint/website/blob/master/src/js/main/index.js#L28

Reply:
Hey, what you have on the current implementation (`algoliaOptions: { facetFilters: [["tags:docs", "tags:blog"]] },`), will behave the same as no `algoliaOptions`, as you only have two tags set in your index and nested arrays will behave as an `OR`.

You can actually keep the same logic but with the `blog` tags.

```suggestion
        facetFilters: ["tags:blog"]
```

Small sandbox with examples: https://codesandbox.io/s/new-frost-7jcxy6?file=/src/index.js

Reply:
I would note that index.js and shortcuts' example use a different app ID than the code in search.js, which goes back to the question I raised - is this a tale of 2 different indexes, one of which is configured correctly while the other isn't?

Reply:
DocSearch defaulted to `BH4D9OD16A` previously, which was a shared app. With our new infra, personal apps are now available and the ESLint one is `L633P0C2IR`